include/config.h

```diff
@@ -5,12 +5,13 @@
  * A catch-all file for configuring various bugfixes and other settings in SM64
  */
 
+#include "config/config_debug.h"
```

Comment:
NOTE: As long as `AUDIO_CRACKLE_DEBUGGING` is in this file, it MUST come before `config_audio.h`